src/app/meshes/index.ts

```diff
@@ -1 +1,25 @@
+import { sources } from './sources'
+import type { ServiceDefinition } from '@/services/utils'
+import { token } from '@/services/utils'
 export * from './routes'
+
+type Token = ReturnType<typeof token>
+type Sources = ReturnType<typeof sources>
+
+const $ = {
+  sources: token<Sources>('mesh.sources'),
+}
+export const services = (app: Record<string, Token>): ServiceDefinition[] => {
+  return [
+    [$.sources, {
+      service: sources,
+      arguments: [
+        app.api,
+      ],
+      labels: [
+        app.sources,
+      ],
+    }],
+  ]
+}
+export const TOKENS = $
```

Comment:
`app/*` things are now becoming proper 'modules'/'plugins' so you can just `build()` up service configuration for infinite amounts of plugins. Importantly you shouldn't have to satisfy/use all 'external' tokens, `undefined` tokens will eventually mean more or less a noop.